src/mococrw/openssl_wrap.h

```diff
@@ -1396,11 +1396,26 @@ std::vector<uint8_t> _EVP_derive_key(const EVP_PKEY *peerkey, const EVP_PKEY *ke
  */
 void _ECDSA_SIG_set0(ECDSA_SIG* sig, SSL_BIGNUM_Ptr r, SSL_BIGNUM_Ptr s);
 
+/**
+ * Get the r signature component as bignum
+ */
+const BIGNUM* _ECDSA_SIG_get0_r(const ECDSA_SIG* sig);
+
+/**
+ * Get the s signature component as bignum
+ */
+const BIGNUM* _ECDSA_SIG_get0_s(const ECDSA_SIG* sig);
+
 /**
  * Get the serialized ECSDA signature in ASN.1 format from ECSDA_SIG object
  */
 std::vector<uint8_t> _i2d_ECSDA_SIG(const ECDSA_SIG*);
```

Comment:
There is still a typo :)

Reply:
thx, found some more typos in comments and exception messages :-D